src/babel.coffee

```diff
@@ -132,7 +132,7 @@ transpile = (sourceCode, filePath, cachePath) ->
 # either generated on the fly or pulled from cache.
 loadFile = (module, filePath) ->
   sourceCode = fs.readFileSync(filePath, 'utf8')
-  unless sourceCode.startsWith('"use 6to5"') or sourceCode.startsWith("'use 6to5'")
+  unless /^("use 6to5"|'use 6to5'|"use babel"|'use babel')/.test(sourceCode)
```

Comment:
I don't know if V8 caches regexes, but in the event that it doesn't, I would extract this regex to a constant so it is not recreated each time.

Also, I don't know how well this performs compared to `startsWith()`.